src/tests/integration/test_kprint.py

```diff
@@ -306,3 +306,57 @@ def test_aliases(self, kprint: KPrint, test_id: str, kast: KInner, expected: str
     def test_nonterm_labels(self, kprint: KPrint, test_id: str, kast: KInner, expected: str) -> None:
         actual = kprint.pretty_print(kast)
         assert actual == expected
+
+
+SORT_COLLECTIONS_TEST_DATA: Iterable[tuple[str, KInner, str]] = (
+    (
+        'single-item',
+        KApply('SetItem', [intToken(1)]),
```

Comment:
You can omit the list for the arguments (for all instances). Also, consider using `token`:

```suggestion
        KApply('SetItem', token(1)),
```

Reply:
We need helpers for these collections in `pyk.prelude`.

Reply:
Done.